nemo/collections/nlp/modules/common/megatron/transformer.py

```diff
@@ -94,47 +114,28 @@ def __init__(
             use_cpu_initialization=use_cpu_initialization,
         )
 
-        if activation in ['geglu', 'reglu', 'swiglu']:
```

Comment:
any specific reason for taking off reglu and swiglu?

Reply:
This looks like a merge issue, I will address that.

Reply:
Done.